teams/teams-ps/teams/Set-TeamsEnvironmentConfig.md

```diff
@@ -0,0 +1,124 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Connect.dll-Help.xml
+Module Name: MicrosoftTeams
+online version:
+schema: 2.0.0
+author: VikneshMSFT
+ms.author: vimohan
+ms.reviewer: pbafna
+manager: vinelap
+---
+
+# Set-TeamsEnvironmentConfig
+
+## SYNOPSIS
+
+Sets environment specific configurations on local machine and will be used to connect to the right environment when running connect-microsoftteams
```

Comment:
would suggest giving an example of the expected usage from Skype for Business On Prem during migration of users from On Prem to Teams online.